### PR TITLE
Add deleting images instructions to dev docs

### LINKED ISSUE #2304: [Images] Delete image using API
### Expected Behavior
There should be a section describing how to delete an image from Cloudflare Images using the API.

### Actual Behavior
The aforementioned section doesn't appear to be present (unless I've missed it!)

### Section that requires update
https://developers.cloudflare.com/images/

### Additional information
I was able to guess the correct API, since it's a simple REST API, and confirm this by watching what happens when deleting an image in the dashboard. However it would be nicer to have this important API detailed in the docs.

### PR DESCRIPTION
**Pull request by rxsoares**
Fixes #2304

### CODE REVIEW

**Comment by rxsoares** [comment]
@marciocloudflare created a PR for your review about the deleting images

File: products/images/src/content/deleting-variants.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Deleting Variants
+order: 5 
+---
+
+# Deleting Variants
+
+You can delete variants via API by using a simple cURL command. 
+
+**Please note;** Deleting a variant is a global action that will afect other images that contain that vartiant.
```
Comment (marciocloudflare) [inline products/images/src/content/deleting-variants.md]:
```suggestion
<Aside type="warning">

Deleting a variant is a global action that will affect other images that contain that variant.

</Aside>

```

Comment (marciocloudflare) [reply products/images/src/content/deleting-variants.md]: This way this warning is visible inside a colored box.

File: products/images/src/content/deleting-images.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Deleting Images
```
Comment (marciocloudflare) [inline products/images/src/content/deleting-images.md]:
```suggestion
```

Comment (marciocloudflare) [reply products/images/src/content/deleting-images.md]: There's no need to have this is the header

File: products/images/src/content/deleting-variants.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: Deleting Variants
```
Comment (marciocloudflare) [inline products/images/src/content/deleting-variants.md]:
```suggestion
```

Comment (marciocloudflare) [reply products/images/src/content/deleting-variants.md]: No need to have this in the header

**Review by marciocloudflare — CHANGES_REQUESTED** [review]
Hi. Can you please check my comments? Thanks

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**